docs/development/decision-records/2024-09-16-proposal-federated-catalog-distribution/README.md

```diff
@@ -0,0 +1,93 @@
+# Proposal for FederatedCatalog with Tractus-X distribution and its TargetNodeDirectory
+
+## Decision
+
+The Federated Catalog will be deployed as a standalone component. The Tractus-X EDC Connector Helm charts will be updated to feature a new Federated Catalog deployment template.
+Regarding the TargetNodeDirectory, a new extension in the FederatedCatalog will expose an API to allow adding participant's identifiers which will be used to obtain the respective data from the Discovery Service.
+
+## Rationale
+
+While a standalone component (= K8S deployment) brings a slight increase in configuration complexity, its ability to be managed and scaled independently makes up for that.
+
+For TargetNodeDirectory it will be set by a new extension responsible for exposing an API, where a member can input the DID's of the participants from which the catalogs are wanted, and then it will retrieve and store the respective Connector URL's. This new extension would get the data from the Discovery Service, and will be named `DiscoveryServiceRetrieverExtension`. This solution allows the member to choose precisely the Target Catalog Nodes that interests them, resulting in reduced network calls and latency.
+Additionally, if a Connector URL is registered (or unregistered) in the Discovery Service, the retriever will reflect it since it requests based on BPN and the registered URL's will be returned.
+
+This solution improves on the default one of having the data in a static file since a dynamic approach would avoid downtime when a change is required.
+
+Other solution for the TargetNodeDirectory was also considered
+- File in a S3 bucket (or different cloud provider's solution)
+  - This solution was discarded due to one file for all instead of each partner having the data that respectively needs does not match the requirement and this solution would lock the usage of a proprietary tool (cloud provider) being harder to sustain in the long run.
```

Comment:
this is more of a solution proposal and thus should be moved to the `"Approach"` section if not already there.

Reply:
I thought that, despite being a suggestion of a solution to be followed and one not be followed this would make sense as a Rationale. But followed that suggestion and transplanted it to the "Approach" section.
Changed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1555/commits/8a3f859f388cbff83e476f248a1952eb8a32ee9e).